setup.cfg

```diff
@@ -34,7 +34,7 @@ install_requires =
     awkward>=2
     numpy>=1.13.3
     vector
-python_requires = >=3.7
+python_requires = >=3.8
```

Comment:
Oh, or it could be this?

Reply:
You need to tell setup-cfg-fmt what your minimum version is. It doesn't seem to pick it up correctly from `python_requires` (though reading the README, it seems like it should - @asottile is something fishy here that needs investigating or does this look expected? It's not the first time I've seen it trying to upgrade this, it seems to happen on every package)

Reply:
oh is python 3.7 officially not supported as of a few days ago or something?

Reply:
Yeah, it's been dead for nine days. https://endoflife.date/python

Though by NEP 29 or SPEC 0 it's been out forever, along with 3.8.

Reply:
That's why you see this downloading package after package scrolling back to find the ones that are compatible with 3.7. :)

Reply:
it's doing the right thing -- if the `--min-py-version` is `>=` what's already set it'll be moved to that

Reply:
There wasn't a `--min-py-version`, this was running with no args at all, default pre-commit hook args only.

Reply:
Ahh, you mean if the default value is `>=` what is already set? Okay.